src/main/java/racing/MovingStrategy.java

```diff
@@ -0,0 +1,5 @@
+package racing;
+
+public interface MovingStrategy {
```

Comment:
이렇게 하나의 메서드만 가지는 인터페이스를  `함수형 인터페이스`라고하는데요.
함수형 인터페이스를 만들고싶은거라면 `@FunctionalInterface` 애너테이션을 붙여주는 것이 좋을 것 같아요. 😉

이 애너테이션을 추가한 뒤, 인터페이스에 임의의 메서드를 하나더 생성해보면 어떻게 되는지 확인해보시고 어떤 현상(?)이 벌어지는지도 코멘트로 간단히 남겨주시면 좋을 것 같네요. 

```suggestion
@FunctionalInterface
public interface MovingStrategy {
```